h/util/document_claims.py

```diff
@@ -262,10 +275,8 @@ def doi_uri_from_string(s):
 
     If the given string doesn't already start with "doi:" it is prepended.
 
-    If the given string does not contain a DOI URI then None is returned.
-    Examples of strings that do not include a DOI are:
-    '', ' ', 'doi:', 'doi: ', ' doi:', ' doi: '.
-
+    If the given string, minus any `doi:` prefix does not match the syntax
+    for DOI names ("10.NNNN/.*") then `None` is returned.
```

Comment:
I think these should be:

    ``...``

not

    `...`

because they're rSt not markdown. We don't actually extract and parse them as rSt anywhere currently though so no biggie.